clang/lib/CIR/CodeGen/CIRGenCall.cpp

```diff
@@ -71,6 +130,23 @@ static cir::CIRCallOpInterface emitCallLikeOp(CIRGenFunction &cgf,
   return builder.createCallOp(callLoc, directFuncOp);
 }
 
+const CIRGenFunctionInfo &
+CIRGenTypes::arrangeFreeFunctionType(CanQual<FunctionProtoType> fpt) {
+  SmallVector<CanQualType, 16> argTypes;
+  for (unsigned i = 0, e = fpt->getNumParams(); i != e; ++i)
```

Comment:
Is this not just a `llvm::copy(fpt->param_types(), std::back_inserter(argTypes))` ?